### PR TITLE
Add HENDRICS

### PR DESCRIPTION
**Pull request by matteobachetti**

### CODE REVIEW

**Comment by matteobachetti** [comment]
As maintainers I added the Stingray ones: @evandromr and @dhuppenkothen

**Comment by matteobachetti** [comment]
@mwcraig All tests passed

File: recipes/hendrics/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  run:
+    - python
+    - astropy
+    - scipy >=0.18.0
+    - numpy >=1.11.0
+    - matplotlib >=1.3
+    - stingray >=0.1rc1
+
+test:
+  imports:
```
Comment (mwcraig) [inline recipes/hendrics/meta.yaml]: The only package you need to import for the test in hendrics

File: recipes/hendrics/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    - astropy
+    - hendrics
+
+about:
+  home: http://github.com/StingraySoftware/stingray
+  license: 3-clause BSD
+  license_family: BSD
+  license_file: LICENSE.rst
+  summary: 'High-ENergy Data Reduction Interface from the Command Shell'
+
```
Comment (mwcraig) [inline recipes/hendrics/meta.yaml]: Another I-wouldn't-mention-if-not-for-the-important-change: Please drop the blank line.

**Comment by matteobachetti** [comment]
@mwcraig done!

**Pull request merged**